src/screens/get_started/index.tsx

```diff
@@ -2,17 +2,43 @@ import React from 'react';
 import Button from '../../components/button';
 import { NavigationInterface } from '../types';
 
-import { Container, Welcome } from './styles';
+import { Container,Toggle_Nav } from './styles';
+import ScreenThree from './screenthree';
 
 interface GetStartedScreenProp extends NavigationInterface {
   testID?: string;
 }
 
 export default function GetStartedScreen(props: GetStartedScreenProp) {
+ 
   return (
     <Container>
-      <Button title="Get started screen button" />
-      <Welcome>Get Started Screen</Welcome>
+      {data.map((item,index)=><ScreenThree 
+      key={index} 
+      header={item.header}
+      title={item.title}
+      description={item.description}
+      />)}
+      <Toggle_Nav>
+        <Button title={'Skip'}  
```

Comment:
```suggestion
        <Button title='Skip'
```
You could actually do it like this since its not a JS variable

Reply:
they are from the branch i merged, Amakri corrected me on that yesterday